### ISSUE TITLE
Setup considerations

### ISSUE THREAD

**Issue by richard-edwards** [IanMercer/imagemultiplier#5]
I think I could put this to good use. Just trying to get it set up optimally right now. What is the best place to put the original assets and the multiplier file. Would a new blank shared app project be the best way to do this?

**Comment by IanMercer** [comment]
Yes, I think a shared 'Artwork' project is a good place for this since it isn't Android or iOS specific.

**Comment by richard-edwards** [comment]
Ok thanks. I've tried that but am having issues with pathing. Are the paths
relative to the solution folder or relative to where the .multiplier file
resides?

On Tue, Apr 14, 2015 at 2:54 PM, Ian Mercer notifications@github.com
wrote:

> Yes, I think a shared 'Artwork' project is a good place for this since it
> isn't Android or iOS specific.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/IanMercer/imagemultiplier/issues/5#issuecomment-93018503
> .

## 

Regards,

Richard

**Comment by IanMercer** [comment]
The source images are relative to the .multiplier file, the outputs are up one directory ("..") from the .multiplier file. I'll add that to the documentation.
